woocommerce/wc-admin
# Add Historical Data Import Screen

Part of #1851.

This PR creates the components for the _Historical Data Import_ section in the _Settings_ screen even though it's not yet connected to the endpoints.

### Screenshots
![image](https://user-images.githubusercontent.com/3616980/56281614-6183af80-610d-11e9-8224-5362d9f216e9.png)

### Detailed test instructions:
- Go to the _Settings_ page.
- Verify there is an _Import Historical Data_ section.
- Try modifying `<HistoricalData>` [props](https://github.com/woocommerce/woocommerce-admin/pull/2073/files#diff-26f12905770396a5868a1a7987751074R165) to simulate different stages of the page.
